docs/faqs.md

```diff
@@ -152,3 +153,27 @@ and [libcurl](https://github.com/conan-io/conan-center-index/blob/f834ee1c825641
 
 However, if logic is too complex (this is subjective and depends on the Conan review team) then just remove the option.
 After one month, we will welcome a PR removing the option that was deprecated.
+
+## Why is `tools.check_min_cppstd call` not enough?
+
+Very often C++ projects require a minimum standard version, such as 14 or 17, in order to compiler. Conan offers tools which enable checking this setting is enabled above this a certain version otherwise it uses the compiler's default.
```

Comment:
I feel that these changes would improve overall clarity:
```suggestion
Very often C++ projects require a minimum standard version, such as 14 or 17, in order to compile. Conan offers tools which enable checking the relevant setting is enabled and above this support for a certain version is present. Otherwise, it uses the compiler's default.
```